gdbgui/backend.py

```diff
@@ -147,6 +147,9 @@ def is_cross_origin(request):
         origin = origin.replace("http://", "")
     elif origin.startswith("https://"):
         origin = origin.replace("https://", "")
+    if host!=origin:
+        logger.warning("Host is Same as Origin")
+        logger.warning("Try using different browser")
```

Comment:
Recommend updating the exist warning
```
logger.warning("Received cross origin request. Aborting")
```
to whatever you think is more descriptive and actionable.

Btw, are you running into this error under normal operation?